ppl/summary/summary.tsx

```diff
@@ -35,12 +36,14 @@ type Tile = {
 }
 
 function getGridLayout(tiles) {
-  return tiles.map((t) => ({i: t.id, ...t.layout}))
+  return tiles.map((t) => ({i: t.id, ...t.layout, static: true}))
```

Comment:
Are we disabling moving of tiles bc we don't want to also support storing that in state right now? or are there also bugs with the actual act of moving them around. I'm also just assuming that's what `static: true` means here.

Reply:
Yes, your assumption is correct on both fronts. We're going to release the first version with no moving or resizing. Then upgrade later.